tests/OperationProcessor.spec.ts

```diff
@@ -40,53 +48,230 @@ async function addBatchOfOneOperation (opBuf: Buffer, cas: Cas, transactionTime:
     batchFileHash: batchFileAddress
   };
 
-  const op = WriteOperation.create(opBuf, resolvedTransaction, 0);
+  const op = WriteOperation.create(opBuf, resolvedTransaction, operationIndex);
   return op;
 }
 
+function getHash (operation: WriteOperation): string {
+  const sha256HashCode = 18;
```

Comment:
As mentioned previously can't hardcode this. Can use the same pattern found in WriteOperation.toDidDcoument().

Reply:
The toDidDocument reference was helpful. Fixed.